reframe/frontend/statistics.py

```diff
@@ -106,11 +107,12 @@ def json(self, force=False):
                     ],
                     'description': check.descr,
                     'display_name': check.display_name,
-                    'filename': inspect.getfile(type(check)),
                     'environment': None,
+                    'filename': inspect.getfile(type(check)),
```

Comment:
I have no problem with the order, i am just making sure. Did you move it in order for them be in alphabetical order? If yes, it should be moved a bit lower, after `fail_reason`.

Reply:
Exactly! I will sort them better ;-)